### DISCUSSION TITLE
No theme option with dark MATE panel and light widgets

### DISCUSSION THREAD

**Discussion by andrewshadura** [ubuntu/yaru#2814] [General]
The GNOME Shell version of the theme supports dark panels and light widgets:
![](https://raw.githubusercontent.com/ubuntu/yaru/hirsute/mate/.github/readme_pics/shell-light.png)
It also works well with Unity.
However, the same look cannot be achieved with MATE: it’s either everything dark or everything light.
I prefer dark panels and titlebars but I don’t want all applications to have dark backgrounds (and especially not websites!), but I cannot achieve that right now.

**Comment by Feichtmeier** [comment]
This was a "design decision" 
Maybe @flexiondotorg can comment on this one and why he decided to do so (I think it was a good decision :) )

**Comment by andrewshadura** [comment]
Well, I’m not sure whether this is a design decision or an oversight, but having three options instead of two shouldn’t be a big deal, especially since the third option exists for other desktop environments.

**Comment by Feichtmeier** [comment]
> Well, I’m not sure whether this is a design decision or an oversight, but having three options instead of two **shouldn’t be a big deal**, especially since the third option exists for other desktop environments.

Yeah, actually it is thouh :smile_cat: 

the gnome desktop with gtk and the shell being two different software technologies is a plus in this context, often is is a minus. so here it is easy to change the one theme without affecting the other.
in mate everything (except the window manager) is gtk, so having a "mixed" theme apart from "light"/"dark" is harder than it might look for you

**Reply by andrewshadura** [reply]
I’m not sure why you closed the issue since while it’s discussable, it still *is* an issue with the theme (since other themes were able to deal with it).

I don’t understand how adding a bunch of CSS rules to make the panel dark is harder — e.g. QogirBudgie has done it and it works very well (I don’t like the colours and some other design decisions of that theme though).

**Reply by andrewshadura** [reply]
E.g. it has:
```css
.mate-panel-menu-bar, .mate-panel-menu-bar menubar,
panel-toplevel.background,
panel-toplevel.background menubar {
  background-color: #21232b;
}
```
in gtk.css. Two other versions (dark and light) of the theme set this to different colours. It’s not that different from what Yaru already does for Unity.
